app/pages/Dashboard.tsx

```diff
@@ -109,33 +108,55 @@ export default function Dashboard() {
         </div>
       </div>
 
-      <div className="mt-6 flex w-full flex-wrap px-10">
-        <div className="w-3/4">
-          <p className="mb-4 text-2xl text-black">Decentralized Identity Verification</p>
-          <p className="text-xl text-black">Select the verification stamps you’d like to connect to your Passport.</p>
-        </div>
+      <div>
+        {viewerConnection.status === "connecting" ? (
+          <div
+            className="absolute right-2 w-full rounded bg-blue-darkblue py-4 px-8 md:right-1/2 md:w-1/4"
+            data-testid="selfId-connection-alert"
+          >
+            <span className="absolute top-0 right-0 flex h-3 w-3 translate-x-1/2 -translate-y-1/2">
+              <span className="absolute inline-flex h-full w-full animate-ping rounded-full bg-green-jade opacity-75"></span>
+              <span className="relative inline-flex h-3 w-3 rounded-full bg-green-jade"></span>
+            </span>
+            <span className="text-green-jade"> Waiting for wallet signature...</span>
+          </div>
+        ) : (
+          <div className="flex w-full flex-wrap p-5 md:hidden md:flex-row">
+            <div className="float-right mb-4 flex flex-row items-center font-medium text-gray-900 md:mb-0">
+              <img src="/assets/gitcoinLogoDark.svg" alt="Gitcoin Logo" />
+              <img className="ml-6 mr-6" src="/assets/logoLine.svg" alt="Logo Line" />
+              <img src="/assets/passportLogoBlack.svg" alt="pPassport Logo" />
+            </div>
```

Comment:
this header bar is repeated on mobile view

also sometimes the `</>` view passport JSON button is shifted down into the next line even if it looks like there's enough space to keep it on the left (because of the `ml-20`?)

![Screen Shot of Gitcoin Passport on iPhone SE size in Chrome inspector](https://user-images.githubusercontent.com/62616399/177360382-03153bf9-29f9-44d4-b9e6-6114a59c312d.png)